### PR TITLE
Feature/header dropdown

### PR DESCRIPTION
**Pull request by PascalUlor**
#### What does this PR do?
Implement logout functionality
implement dropdown
#### Any background context you want to provide?

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/store/user/sagas.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { all, call, takeLatest, put, select } from "redux-saga/effects";
-import { UserTypes, setUser, userError, resetUser, setUserProfile } from "./actions";
+import { persistStore } from 'redux-persist';
```
Comment (attaradev) [inline src/store/user/sagas.js]:
```suggestion
```

File: src/store/user/sagas.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+import { store } from '../../store';
```
Comment (attaradev) [inline src/store/user/sagas.js]:
```suggestion
import { persistor } from '../../store';
```

File: src/store/user/sagas.js [hunk 3]
```diff
@@ -62,11 +64,16 @@ function* watchSocialAuth() {
 }
 
 function* logout() {
-  yield put(resetUser());
+  try {
+    const persistor = persistStore(store)
```
Comment (attaradev) [inline src/store/user/sagas.js]:
```suggestion
```

**Review by attaradev — CHANGES_REQUESTED** [review]

File: src/store/user/sagas.js (hunk 3 above)
Comment (PascalUlor) [reply src/store/user/sagas.js]: done

**Pull request merged**